manifests/generated/vmirs-resource.yaml

```diff
@@ -259,6 +259,16 @@ spec:
                               persistentVolumeClaim: {}
                           name:
                             type: string
+                          networkDisk:
```

Comment:
Here i'd follow the pattern of struct per type. Ie

```
networkDisk:
  http:
    ...
```

This would allow us to extend it with i.e. rbd, iscsi,... Easily.

Reply:
Ah, that makes sense; easier than bolting things on via the Protocol parameter I added.  I'll get an update to that later today or first thing in the morning. Thanks!